discovery-infra/tests/base_test.py

```diff
@@ -1,32 +1,33 @@
 import os
 import pytest
 
-import assisted_service_api
 import utils
 import consts
+import assisted_service_api
 from tests.conftest import env_variables
 
 if os.environ.get('NODE_ENV') == 'QE_VM':
-    from node_controlers.qe_vm_controler import QeVmController as node_controler
+    from node_controlers.qe_vm_controler import QeVmController as nodeController
 else:
     raise NotImplementedError
 
+
 class BaseTest:
```

Comment:
Now that the functions are static methods I don't see a reason to have a class.
All these methods can be public as well without a class.